client/codegen_pinvoke/Main.cs

```diff
@@ -161,4 +170,22 @@ public static DirectoryInfo GetProjectDir()
     }
 
 
+    public class VisitMethodPass_IgnoreCopyConstructor : TranslationUnitPass

```

Comment:
Add a class docstring that explains what this does

Reply:
Adding.